### PR TITLE
Update SearchBar.js for RN 0.25 / ES6

### PR DESCRIPTION
**Pull request by Mokto**
#74 is not enough IMO as React users now use ECMA6.

### CODE REVIEW

File: SearchBar.js [hunk 1]
```diff
@@ -1,15 +1,11 @@
-var NativeModules, PropTypes, RNSearchBar, React, SearchBar;
+import { Component, PropTypes } from 'react';
+import React, { NativeModules, requireNativeComponent } from 'react-native';
```
Comment (alaycock) [inline SearchBar.js]: As per the [0.25.1 release notes](https://github.com/facebook/react-native/releases/tag/v0.25.1), I think you should import `React` from the `'react'` module. The code above will still work, but is deprecated functionality.

Comment (Mokto) [inline SearchBar.js]: I should maybe call the variable import ReactNative from 'react-native';

But I need import from 'react-native' to use ReactNative.findNodeHandle.

BTW I tested it on RN 0.25 and it works without warnings.

Comment (alaycock) [inline SearchBar.js]: Oops, my mistake, but you're right, the name ReactNative might be more appropriate. 👍

**Comment by umhan35** [comment]
Is ES6 required?

SearchBar.js is generated from SearchBar.coffee.

**Comment by Mokto** [comment]
I think if you want :
1) this plugin maintained
2) help from other developers

This is a logical choice.

But this is your call ! If you can take on this project all by yourself, you decide ! I like what you did with this plugin, but as I said it [here](https://github.com/umhan35/react-native-search-bar/issues/76), I think you have to let people help you. There are 15 issues and 8 PR going on.

**Comment by umhan35** [comment]
@Mokto Could you answer my previous question?

I answer you last question in the issue you opened.

**Comment by Mokto** [comment]
Not it is not required ;)

**Comment by umhan35** [comment]
...

#74 is merged and is enough.

**Comment by Mokto** [comment]
Ok thanks !

**Pull request closed without merging**